app/src/main/java/com/example/quizacademytask/FlashcardStackFragment.kt

```diff
@@ -41,7 +41,6 @@ class FlashcardStackFragment : Fragment() {
 
         //Set the toolbar
         toolbar = binding.toolbarFlashcardStack
-        requireActivity().setActionBar(toolbar)
         toolbar.setNavigationOnClickListener { requireActivity().onBackPressed() }
```

Comment:
this can be handled in on onOptionsItemSelected by reacting to android.R.id.home

Reply:
```kotlin
override fun onOptionsItemSelected(item: MenuItem): Boolean {
        when (item.itemId) {

            android.R.id.home -> {
```